src/dual_model_variables.jl

```diff
@@ -176,9 +178,14 @@ function set_dual_variable_name(
     i::Int,
     ci_name::String,
     prefix::String,
+    unique_var::Bool = false,
```

Comment:
We don't need a default value if we set this value everywhere we use this function. Otherwise that just adds a chance to forget to set this and have a silent bug

Reply:
was to avoid breaking things but fine with no default